lib/util.js

```diff
@@ -371,10 +371,11 @@ function formatValue(ctx, value, recurseTimes) {
   // Look up the keys of the object.
   var keys = Object.keys(value);
   var visibleKeys = arrayToHash(keys);
```

Comment:
I don't fully understand what goes on with `keys` vs. `visibleKeys`, yet, this seems to work.